CHANGELOG.md

```diff
@@ -46,4 +46,4 @@ _None._
 
 ### Internal Changes
 
-_None._
+- Add this changelog file. [#545]
```

Comment:
I debated wether to use the full URL, which is more friendly to being used outside of GitHub, where the rendered wouldn't expand `#number` to a PR URL.

```suggestion
- Add this changelog file. [#545](https://github.com/wordpress-mobile/WordPressKit-iOS/pull/545)
```

I decided against it because:

- We don't do that in other projects
- Using the number alone is simpler
- What are the chances of actually needing these raw changelog notes elsewhere? (_If we'll ever need them, it's straightforward to add a parsing step to introduce the full URLs ourselves_).

Reply:
Yeah I remember similarly debating about that when I started the `CHANGELOG.md` in `release-toolkit`. And sometimes I miss the fact that those are not clickable (while in `RELEASE-NOTES.txt` file we use in our client app's projects, contributors do add the URL).

But ultimately I agree that it's simpler to just add the PR number alone, reduces the risk of accidental inconsistency (with a bad copy/paste where we'd change the PR number but forgot to update the URL…). And in the seldom cases where we do need to look at the associated PR for a changelog entry, which is not that frequent to be a burden imho, it's not that much effort to enter the URL manually in your browser, so simpler to write wins for that case in my book.